opm/parser/eclipse/EclipseState/Tables/TableManager.hpp

```diff
@@ -213,6 +215,72 @@ namespace Opm {
         double rtemp() const;
 
         bool operator==(const TableManager& data) const;
+
+        template<class Serializer>
+        void serializeOp(Serializer& serializer)
+        {
+            auto simpleTables = m_simpleTables;
```

Comment:
the copy might be expensive. Maybe in the long run we can tell map() to exclude certain keys from serializing?

Reply:
maybe. i don't think it's worth it for this as it's a map from a string to what is basically a map from size_t to shared_ptr.